IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcRelInterferesElements/DocEntity.xml

```diff
@@ -15,10 +15,10 @@
 			<Documentation></Documentation>
 		</DocAttribute>
 		<DocAttribute Name="InterferenceType" UniqueId="a92075e8-181d-4233-bd6a-e3efed6da830" DefinedType="IfcIdentifier" AttributeFlags="1">
-			<Documentation>Optional identifier that describes the nature of the interference. Examples include  &apos;&apos;Clash&apos;&apos;, &apos;&apos;ProvisionForVoid&apos;&apos; (physical elements), and �Crosses�, �PassesThrough� �PassesOver� �PassesUnder� (spatial elements).</Documentation>
+			<Documentation>Optional identifier that describes the nature of the interference, see details in documentation.</Documentation>
```

Comment:
I would remove the 'see details in documentation' here. This should be obvious.

Reply:
Done